x-pack/plugins/fleet/public/hooks/use_request/uninstall_commands.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { uninstallTokensRouteService } from '../../../common/services';
+
+import type {
+  GetUninstallTokensRequest,
+  GetUninstallTokensResponse,
+} from '../../../common/types/rest_spec/uninstall_token';
+
+import { useRequest } from './use_request';
+
+export const useGetUninstallTokens = ({
+  policyId,
+  page,
+  perPage,
+}: GetUninstallTokensRequest['query'] = {}) => {
+  return useRequest<GetUninstallTokensResponse>({
+    method: 'get',
+    path: uninstallTokensRouteService.getListPath(),
+    query: {
+      policyId,
+      page,
+      perPage,
+    } as GetUninstallTokensRequest['query'],
```

Comment:
Instead of doing this casting, could you define a const above with these props?

Reply:
sure, that's cleaner, thanks!
4afe522fc8c2aeff5c3a1c5d0f416645d8b76810